### DISCUSSION TITLE
Feature Request - Allow for multiple paginations in one livewire component.

### DISCUSSION THREAD

**Discussion by FoksVHox** [livewire/livewire#1833] [3. Feature ideas] [answered]
The title says all.

**Comment by austenc** [comment, marked as answer]
This is a duplicate of https://github.com/livewire/livewire/issues/1209

**Reply by ju5t** [reply]
It is and isn't. This title says a single component should support it whereas the issue discusses pagination in two components. Having two components makes more sense to me as it is much simpler. In that case, #1209 is 👍

**Reply by FoksVHox** [reply]
The goal of this is to be able to have different pagination like so:
![image](https://user-images.githubusercontent.com/37844753/96334366-93fac980-1070-11eb-82f9-c196e346054b.png)

**Reply by nuernbergerA** [reply]
Just split the sections into their own livewire components.

**Reply by FoksVHox** [reply]
Works like a charm, thanks @nuernbergerA
